### ISSUE TITLE
Encode only binary or decode with key/password binary

### ISSUE THREAD

**Issue by ggagnidze** [OOPS-ORG-PHP/mod_screwim#18]
**Is your feature request related to a problem? Please describe:**

For example I have several servers, installed on the client site and development is going right on them.
I want to encode files on this servers but not decode them.

**Describe the solution you'd like:**
There should be encode only binary, so I can run it and encode file with no parameters passed.
Or better way: binary, which will encode without additional params, but Decoding must ask for a key / password.

**Describe alternatives you've considered:**

For now I can't use the same binary to encode, because it can be also used to decode with no problem.

**Additional context:**

Basically new binary should copy behaviour of api. Code anything, decode to file — only with key / password (and key / password must not be given as parameter, because shell history can store it).

**Comment by Joungkyun** [comment]
I'm sorry .. I do not understand what the situation explains.

**Comment by Joungkyun** [comment]
do you want the option ```--decode``` to disappear?

```bash
[root@host tools]$ /usr/bin/screwim -h
screwim 1.0.5 : encode or decode php file
Usage: screwim [OPTION] PHP_FILE
   -c VAL, --convert=VAL convert key byte to digits
   -d,     --decode   decrypt encrypted php script --------------> do you want to remove?
   -h,     --help     this help messages
   -H VAL, --hlen=VAL length of magic key(SCREWIM_LEN or PM9SCREW_LEN). use with -d mode
   -k VAL, --key=VAL  key bytes. use with -d mode
   -v,     --view     print head length and key byte of this file
```

**Comment by Joungkyun** [comment]
There is no problem providing binary that does not support decoding. However, what you want is probably as follows.

1. It must be encrypted without any command line arguments. (Because of ```.bash_history```)
2. When decrypting, a key string must be received. But don't leave it in the ```.bash_history```.

The first issue is originally supported, so there is no problem.

However, the second issue is ambiguous.

screwim cannot control leaving an execution history in the .bash_history. It depends on the user's behavior as command ```export HISTFLIE=/dev/null```.
Therefore, the problem of decoding by giving a key string as an option is still dangerous.

Finally, the condition you want seems to mean that you will not leave the original file.
This is a very dangerous idea.

Decryption is an additional function, not a function that supports recovery if there is no original. For stronger security, it is recommended to back up the original file in a separate location rather than attempting decryption.
Therefore, if run ```./configure``` with the ```--disable-decode``` option, you will get the ```screwim``` command without decryption.
